tasks/deploy_app.yml

```diff
@@ -19,13 +19,35 @@
     that:
       - tc_home is defined
       - tc_home.stat is defined
-      - tc_home.stat.exists
-      - tc_home.stat.isdir 
     quiet: true
 
+- name: Create webapps directory for the application
```

Comment:
This should be done by the collection. You could do an assert: that check that the webapps folder exists, but, honestly, I would not bother and simply remove this part of the playbook.